src/Sylius/Bundle/ShopBundle/ShopSession.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle;
+
+final class ShopSession
+{
+    public const SESSION_NAME = '_sylius.cart';
```

Comment:
In fact it isn't a session name, but a key under a cart id is stored in session. There are a few more places that use `_sylius.cart`, let's change either all of them or none.

Another thought: why wouldn't we make it a service container parameter? This way end users will be able to change it in their apps.